src/screens/ReceiveFundsSelectAccount.js

```diff
@@ -36,13 +36,9 @@ class ReceiveFundsSelectAccount extends Component<{
 
   onAccountPress = (account: Account) => {
     const { navigation } = this.props;
-    const {
-      selectedAccountId,
-      setAccountId,
-      syncCounterValue
-    } = navigation.state.params;
-    if (selectedAccountId !== account.id) setAccountId(account.id);
-    syncCounterValue(account.unit.code);
+    const { selectedAccountId, setSelectedAccount } = navigation.state.params;
+    if (selectedAccountId !== account.id)
+      setSelectedAccount(account.id, account.unit.code);
```

Comment:
to keep things the most canonical possible, I think it should just be setting back the accountId like before, but only the other side would have to find the account by id again to find what it its code.

> On a side note, ideally, we should find a better way to communicate back to the parent navigation. the fact we give a callback in the navigate is still a hack to me but I have not found a better quick solution for it. It was suggested by some folks of react-navigation but I feel there are some design problem with it (a callback is not serializable / can't recreate the screen stack if things are connected with a cb). I would dream if it was possible to do `navigation.goBackWithAnArgument()` and the previous parent screen would handle it

(also, one other thing to note is that unit.code is not really reliable & we'll introduce currency.ticker later)